### PR TITLE
Update visual of login page

### LINKED ISSUE #178: Login: Update login page
### Describe the feature

Comparing our figma's login page with our current one clearly shows that the one currently in place is but a place-holder, not matching the website's structure and theme. It would be great to keep the theme going and fix this problem.

### Use Case

- When I use the login page, I want to not feel like I am interacting with a prototype, so that I feel safe about it.

### Proposed Solution

Use figma's login design that was created for this page.

### Other Information

_No response_

### PR DESCRIPTION
**Pull request by fungame2270**
### Issue

Closes #178 .

### Reason for this change

This change updates the ui for the new design on figma

### Description of changes

Added login screen to root added particles for background

### Description of how you validated changes

The changes are visualy checked comparing to figma

### Checklist
- [X] I have tested my changes locally.
- [ ] I have updated the documentation to reflect my changes.
- [ ] I have added/updated unit tests (if applicable).
- [X] My code adheres to the [CONTRIBUTING GUIDE](https://ua-smart-signage-platform.github.io/Documentation-Website/guidelines/)

### CODE REVIEW

File: frontend/src/routes/Wso2Login/index.jsx [hunk 1]
```diff
@@ -21,10 +25,22 @@ const setWithExpiry = (key, value, ttl) => {
...
+    initParticlesEngine(async (engine) => {
+      await loadFull(engine);
+    }).then(() => {
+      setInit(true);
+    });
+  }, []);
+
+  const particlesLoaded = (container) => {
+    console.log(container);
+  };
```
Comment (pdPinho) [inline frontend/src/routes/Wso2Login/index.jsx]: Unnecessary code perhaps?

File: frontend/src/routes/Wso2Login/index.jsx [hunk 2]
```diff
@@ -53,51 +69,30 @@ const Wso2Login = () => {
...
-    </div>
+  const options = useMemo(
+    ()=> {
+      if(nyan){
+        return nyanParticles
+      }else{
+        return greenParticles
+      }
+    },
+    [nyan],
```
Comment (pdPinho) [inline frontend/src/routes/Wso2Login/index.jsx]: scary styling format compared to the rest

File: frontend/src/routes/Wso2Login/SignInDiv.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            </button>
+          </div>
+          : 
+          <div className='flex flex-col gap-6 items-center justify-center'>
+            <button className='bg-primary font-bold text-xl p-3 rounded-md px-16'>Sign in with IDP</button>
+            <button className='bg-[#95A967] font-bold text-lg p-1 rounded-md w-[75%]'
+                    onMouseDown={()=> setLoginRoot(!loginRoot)}>
+                    Admin Sign in
+            </button>
+          </div>
```
Comment (pdPinho) [inline frontend/src/routes/Wso2Login/SignInDiv.jsx]: Want to take this into a const and then simply call it?

**Review by pdPinho — CHANGES_REQUESTED** [review]
Check comments and fix the appropriate ones.

File: frontend/src/routes/Wso2Login/index.jsx (hunk 1 above)
Comment (fungame2270) [reply frontend/src/routes/Wso2Login/index.jsx]: i dont know

**Review by pdPinho — APPROVED** [review]
Looks good to me 👍

**Review by pdPinho — APPROVED** [review]
LGTM 👍

**Pull request merged**